src/ui/Analytics/AnalyticsActionListMeta.ts

```diff
@@ -84,6 +84,9 @@ export interface IAnalyticsFacetMeta {
   facetId: string;
   facetField: string;
   facetValue?: string;
+  facetRangeStart?: string;
```

Comment:
Rather than adding optional properties to this interface, the better practice is to create a new one extending `IAnalyticsFacetMeta`. There are several benefits that come from doing this:

1. Remove run-time conditionals checking whether a property exists or not.
2. Leverage typescript better by saying a collection of properties should exist and be defined together. e.g.

```
interface IAnalyticsRangeFacetMeta {
    facetRangeStart: string;
    facetRangeEnd: string;
    facetRangeEndInclusive: boolean;
}
```

Reply:
This could be done In a refactor (the same as splitting the values)